src/apps/vpn/ui/screens/ScreenRemovingDevice.qml

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.15
+
+import Mozilla.Shared 1.0
+import Mozilla.VPN 1.0
+import components 0.1
+
+MZLoader {
+    objectName: "removingDevice"
+    footerLinkIsVisible: false
+    headlineText: MZI18n.DevicesRemovingTitle
+
+    Timer {
+        running: true
+        interval: 10000
+        onTriggered: cancelBtn.visible = true
```

Comment:
This came out of a discussion with Santiago and Peter:

It's because in most cases, the task will complete fast enough so that the user never sees it and never has the opportunity to go back. Only if the task is taking a long time (> 10 seconds) should we offer them the ability to go back since something would appear to be going wrong.